crates/matrix-sdk-ui/src/room_list_service/filters/favourite.rs

```diff
@@ -0,0 +1,82 @@
+use matrix_sdk::{Client, RoomListEntry};
```

Comment:
Do we need the copyright header here too?

Reply:
I'm not sure it's mandatory accordingly to my last discussion with the team internally. I'm adding them in case of.